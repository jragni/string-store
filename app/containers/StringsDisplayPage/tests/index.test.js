/**
 * Test for the StringDisplayPage
 */

import React from 'react';
import { render } from 'react-testing-library';
import { IntlProvider } from 'react-intl';
import { Provider } from 'react-redux';
import { browserHistory } from 'react-router-dom';

import { StringsDisplayPage, mapDispatchToProps } from '../index';
// import { loadStrings } from '../../App/actions';
import configureStore from '../../../configureStore';

describe('<StringDisplayPages />', () => {
  let store;
  const onMount = jest.fn();

  beforeAll(() => {
    store = configureStore({}, browserHistory);
  });

  it('should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(
      <Provider store={store}>
        <IntlProvider locale="en">
          <StringsDisplayPage
            onMount={onMount}
            loading={false}
            error={false}
            string={[]}
          />
        </IntlProvider>
      </Provider>,
    );
    expect(firstChild).toMatchSnapshot();
  });
});
