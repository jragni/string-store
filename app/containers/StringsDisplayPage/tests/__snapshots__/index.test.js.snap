// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StringDisplayPages /> should render and match the snapshot 1`] = `
<div>
  <h1
    class="H1-tncdg6-0 hpAKSC"
  >
    <span>
      String Store Strings
    </span>
  </h1>
  <div
    class="Wrapper-sc-1umgotm-0 laCwDT"
  >
    <ul
      class="Ul-mwnq6h-0 jcxXFy"
    />
  </div>
</div>
`;
